### ISSUE TITLE
Issue on choosing question type as 'rating'

### ISSUE THREAD

**Issue by Matwog** [surveyjs/survey-library#2247]
### Are you requesting a feature, reporting a bug or asking a question?
Reporting a bug

### What is the current behavior?

- The styling for the rating type "radio field" is taking the styling class
```
 .sv_main .sv-visuallyhidden {
      position: absolute;
    clip: rect(1px, 1px, 1px, 1px);
    height: 1px !important;
    width: 1px !important;
    overflow: hidden;

```
Due to this the radio buttons are not visible.

- On trying to add custom styling it is observed that before the minRateDescription a radio button is being displayed. 
```
<div>
   <label>
      <span>minRateDescription</span>
      <input type='radio' />
      <span>1</span>
   </label>
</div>
```
But the html structure that is currently being rendered is  
```
<div>
   <label>
      <input type='radio' />
      <span>minRateDescription</span>
      <span>1</span>
   </label>
</div>
```

### What is the expected behavior?
```
<div>
   <label>
      <span>minRateDescription</span>
      <input type='radio' />
      <span>1</span>
   </label>
</div>
```

### How would you reproduce the current behavior (if this is a bug)?
The bug is reproducible by adding a question of typing rating.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
const q6: Questions = {
    type: "rating",
    name: "satisfaction",
    title: "Below you will find word pairs that you can use to rate the survey. They each represent extreme opposites, between which a gradation is possible.",
    isRequired: true,
    minRateDescription: "Unsympathetic",
    maxRateDescription: "Sympathetic"
}

```

### Specify your

* browser: chrome
* browser version: 83.0
* surveyjs platform (angular or react or jquery or knockout or vue): survey-react
* surveyjs version: 1.7.15

**Comment by tsv2013** [comment]
Here is the live sample with your JSON - https://plnkr.co/edit/b7FK20CY02cK76ln
It works ok for me. What am I missing?

**Comment by Matwog** [comment]
The bug is reproducible by commenting out the use of modern.css.

```
<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Ask a user to rate a something by choosing a value - Rating question, jQuery Survey Library Example</title>

        <meta name="viewport" content="width=device-width"/>
        <script src="https://unpkg.com/jquery"></script>
        <script src="https://surveyjs.azureedge.net/1.7.15/survey.jquery.js"></script>
        <!-- <link href="https://surveyjs.azureedge.net/1.7.15/modern.css" type="text/css" rel="stylesheet"/> -->
        <link rel="stylesheet" href="./index.css">

    </head>
    <body>

        <div id="surveyElement" style="display:inline-block;width:100%;"></div>
        <div id="surveyResult"></div>

        <script type="text/javascript" src="./index.js"></script>

    </body>
</html>
```

**Comment by tsv2013** [comment]
If you don't want to use the modern theme and don't load corresponding CSS file, you need to set another theme (the "default" one I guess):
```JS
Survey
    .StylesManager
    .applyTheme("default");
```

**Comment by Matwog** [comment]
So what basically the styling is doing is that it tries to keep the radio button barely visible so the user has to click on the corresponding span element to make the selection. But for my use case I tried to provide with radio buttons. So when I try to override the styling to make the radio buttons visible I noticed that the  a radio button is above the minRateDescription in the html tree. So thought it was a bug. 

But it would be nice to have this structure 
```
<div>
   <label>
      <span>minRateDescription</span>
      <input type='radio' />
      <span>1</span>
   </label>
</div>
```
instead of this 

```
<div>
   <label>
      <input type='radio' />
      <span>minRateDescription</span>
      <span>1</span>
   </label>
</div>
```

If the team feels that this is something that will be fixed sometime in the future then its great. I will try to customise the css to fit my use case. Thanks guys for **survey-react**. It is a great library. :)
